TACC/Core-CMS
# Task/tup 395 cmd news layout switch

## Overview

Add (hidden) form to swap News list layout (list, grid).

## Related

- early prep for follow-up to [TUP-395](https://jira.tacc.utexas.edu/browse/TUP-395)
- early prep for follow-up to TACC/Core-CMS#587

## Changes

- add layout switch form template
- add layout switch module script

## Testing

0. Load article listing.
1. Use Developer tools to "Inspect Element".
2. Unhide the `<form id="layout-choice">`
3. Click List (≡) radio button.
4. Verify the `<section class="blog-list">` becomes `<section class="blog-list as-list">`.
5. Click Grid (⸬) radio button.
6. Verify the `<section class="blog-list">` becomes `<section class="blog-list as-grid">`.

## UI

| hidden | un-hidden |
| - | - |
| ![hidden](https://user-images.githubusercontent.com/62723358/216481122-be197f1b-0c57-4e72-acc6-30702a91ce04.png) | ![unhidden](https://user-images.githubusercontent.com/62723358/216481120-b6faedbc-6b49-4ac8-8f86-33790210d64d.png) |

| as-grid | as-list |
| - | - |
| ![as-grid](https://user-images.githubusercontent.com/62723358/216481114-8e3e92fb-231e-4ce1-8a9d-c62560fbde0a.png) | ![as-list](https://user-images.githubusercontent.com/62723358/216481117-378546d1-124d-4e82-b136-506692143d09.png) |
